### ISSUE TITLE
code for parse_addr not too correct

### ISSUE THREAD

**Issue by tshepang** [achanda/ipnetwork#62]
The following should fail to parse:

```rust
let net: Ipv4Network = "1/24".parse()
println!("{}", net);
```

We instead get:

> 1.0.0.0/24

**Comment by achanda** [comment]
Thanks for reporting these. Unfortunately, I am on vacation and will not have much time to take a look till after 10th October. Do feel free to send a PR in the mean time!

**Comment by tshepang** [comment]
Ok, looking at tests, it does look like this a valid way of specifying the network. Please close if I'm correct.

**Comment by achanda** [comment]
This does look like a bug and should be fixed. Marking it for now.

**Comment by sharksforarms** [comment]
Depends who you ask:
http://jodies.de/ipcalc?host=1&mask1=32&mask2=

Address: 1
Netmask: 24
```
Address:   1.0.0.0               00000001.00000000.00000000.00000000 
Netmask:   255.255.255.255 = 32  11111111.11111111.11111111.11111111 
Wildcard:  0.0.0.0               00000000.00000000.00000000.00000000 
=>
Network:   1.0.0.0/32            00000001.00000000.00000000.00000000  (Class A)
Broadcast: 1.0.0.0               00000001.00000000.00000000.00000000 
HostMin:   1.0.0.1               00000001.00000000.00000000.00000001 
HostMax:   1.0.0.0               00000001.00000000.00000000.00000000 
Hosts/Net: 1                
```

**Comment by achanda** [comment]
Python and Ruby both rejected it though
```
$ python3
Python 3.6.1 |Anaconda custom (64-bit)| (default, May 11 2017, 13:04:09)
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import ipaddress
>>> ipaddress.ip_network('1/24')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/Abhishek/anaconda3/lib/python3.6/ipaddress.py", line 84, in ip_network
    address)
ValueError: '1/24' does not appear to be an IPv4 or IPv6 network
>>>
$ irb
irb(main):001:0> require 'ipaddr'
=> true
irb(main):002:0> IPAddr.new "1/24"
Traceback (most recent call last):
        5: from /usr/local/bin/irb:11:in `<main>'
        4: from (irb):2
        3: from (irb):2:in `new'
        2: from /usr/local/Cellar/ruby/2.5.1/lib/ruby/2.5.0/ipaddr.rb:586:in `initialize'
        1: from /usr/local/Cellar/ruby/2.5.1/lib/ruby/2.5.0/ipaddr.rb:649:in `in6_addr'
IPAddr::InvalidAddressError (invalid address)
```

**Comment by sharksforarms** [comment]
If the decision is not to support it, it can offload the parsing to the standard modules:
https://doc.rust-lang.org/std/net/struct.Ipv4Addr.html
https://doc.rust-lang.org/std/net/struct.Ipv6Addr.html

**Comment by stusmall** [comment]
@achanda Was this included in the 0.13.0 release?  It looks like it might not be.  If not can we get a 0.13.1 release?

**Comment by achanda** [comment]
@stusmall published just now!
